docs/reference/query.md

```diff
@@ -616,31 +616,26 @@ const result = await client.request(
 
 ## Deep
 
-Deep allows you to set any of the other query parameters on a nested relational dataset.
+Deep allows you to set any of the other query parameters (expect for [Fields](#fields) and [Deep](#deep) itself) on a
+nested relational dataset.
 
-### Examples
-
-Limit the nested related articles to 3
+The nested query parameters are to be prefixed with an underscore.
 
-```json
-{
-	"related_articles": {
-		"_limit": 3
-	}
-}
-```
+### Examples
 
-Only get 3 related articles, with only the top rated comment nested
+Only get 3 related posts, with only the top rated comment nested:
 
 ```json
 {
-	"related_articles": {
-		"_limit": 3,
-		"comments": {
-			"_sort": "rating",
-			"_limit": 1
+	"deep": {
```

Comment:
Do we want to specify the difference here between using the API and the SDK?

Because the API can be either:
```
?deep[related_posts][_limit]=3
```
or 
```
?deep={"related_posts":{"_limit": 3}}
```

and the updated example would only be correct in the SDK context of:
```
await directus.request(readItems('collection', {
  "deep": {
    "related_posts": {
      "_limit": 3,
    }
  }
}));
```

🤔

Reply:
I think this should already made clear beneath with the specific examples:

https://github.com/directus/directus/blob/1124171da5b6858faeade1ca67d12386c3c688a9/docs/reference/query.md?plain=1#L642-L695

Note: I wouldn't mind dropping the generic example altogether (in favor of adding some descriptions to the specific examples) if that eliminates the confusion.

Reply:
Thats fair i was looking much at the context surrounding the example